src/main/java/chalmers/pimp/model/canvas/layer/LayerDelegate.java

```diff
@@ -13,15 +13,17 @@
    * The default value for the visibility property.
    */
   static final boolean DEFAULT_VISIBILITY_VALUE = true;
-
+  static int layerIndex = 1;
```

Comment:
Bad solution.

Reply:
I'm open for suggestions, thought it would be better to always have a new number rather than the Layers having the same name.